test/test_distributions.py

```diff
@@ -293,6 +297,47 @@ def test_bernoulli_3d(self):
                          (2, 5, 2, 3, 5))
         self.assertEqual(Bernoulli(p).sample_n(2).size(), (2, 2, 3, 5))
 
+    def test_multinomial_1d(self):
+        p = Variable(torch.Tensor([0.1, 0.2, 0.3]), requires_grad=True)
+        self.assertEqual(Multinomial(10, p).sample().size(), (3,))
+        self.assertEqual(Multinomial(10, p).sample((2, 2)).size(), (2, 2, 3))
+        self.assertEqual(Multinomial(10, p).sample_n(1).size(), (1, 3))
+        self._gradcheck_log_prob(Multinomial, (10, p))
+        self._gradcheck_log_prob(Multinomial, (10, None, p.log()))
```

Comment:
@fritzo Am I calling this this the wrong way? No idea why it fails...

Reply:
I think `gradcheck()` may be trying to differentiate WRT `total_count`. You could try hard binding the 10 to `Multinomial`:
```py
self._gradcheck_log_prob(lambda p: Multinomial(10, p), [p])
```